### DISCUSSION TITLE
resign can't find signing identity

### DISCUSSION THREAD

**Discussion by sel38582** [fastlane/fastlane#17541] [General]
Hi all,  just a request regarding the resign of an ipa files using enterprise certificate.
The distribution certificate is hosted on a github  repository and so I'm using match in fastfile for retrieving the certificate and installing it in a temp keychain (using circleci mac custom image). The certificate is well retrieve and installed, the provisioning profile also, and when resign is run from fastfile or shell command '''fastlane sigh resign com.factory.myapp.ipa --signing_identity ""iPhone Distribution: MyFactory" --provisioning_profile "/Users/distiller/Library/MobileDevice/Provisioning Profiles/4ffa09d4-1212-1212-1212-12121121295.mobileprovision" --keychain_path "/Users/distiller/Library/Keychains/fastlane_tmp_keychain-db''' I got the error that signing Identity could not be find. I f I ssh to the instance and do security find-certificate it could find it so well installed. attached is console result of instance, signing identity is empty ...
[cci_fastlane_resign_failed.txt](https://github.com/fastlane/fastlane/files/5477555/cci_fastlane_resign_failed.txt)

**Comment by rogerluan** [comment]
@sel38582  please file a bug report [here](https://github.com/fastlane/fastlane/issues/new/choose) instead.
